local_server/local-server-services/README.md

```diff
@@ -1,38 +1,55 @@
-# Local Server Services
+# Local server services
 
-Manage multiple running services in Local Server.
+Demonstrates how to start and stop the Local Server and start and stop a local map, feature, and geoprocessing service running on the Local Server.
 
-**Note:** Local Server is not supported on MacOS
+![Image of local server services](LocalServerServices.png)
 
-![](LocalServerServices.png)
+## Use case
+
+For executing offline geoprocessing tasks in your ArcGIS Runtime apps via an offline (local) server.
 
 ## How to use the sample
 
-Choose an option from the dropdown control to filter packages by service type. Then click the Open button to choose a package. Finally, click the Start button to start the service. The service's status will be displayed in the center log.
+Click `Start Local Server` to start the Local Server. Click `Stop Local Server` to stop the Local Server.
```

Comment:
I don't have local server up and running to check how it looks, but we should probably stick with the original description of how to use the sample which will be Java specific

> Choose an option from the dropdown control to filter packages by service type. Then click the Open button to choose a package. Finally, click the Start button to start the service. The service's status will be displayed in the center log.

> To stop a service, select it from the Running Services list and click the Stop Service button. To go to the service's URL in your default web browser, select it and click the Go to URL button.